src/constants/routes.constants.coffee

```diff
@@ -4,7 +4,7 @@ routes_Names =
   components: {}
   views:
     # defined via dedicated state: '/user/article'
-    guest     : ['about',  'features', 'home', 'login', 'pwd_forgot', 'pwd_reset', 'sign_up']
+    guest     : ['about',  'features', 'home', 'login', 'pwd_forgot', 'sign_up']
```

Comment:
the view it is not part of the constants since we need a route that matches the url pattern (i.e /username/token)